### PR TITLE
Band-aids closets spawning their items twice and adds some debug info to try and catch the bug in the act

### LINKED ISSUE #28744: Synergy Station Hos Locker
(WEB REPORT BY: burntdevil REMOTE: 206.221.180.138:7777)
# Revision
c0478ace14ebce2dfa85382a976b8cb1dfc90a07

# Description

The entire contents is duplicated. They get two of everything.

# Steps to Reproduce
open locker
# What you Expected
get one debit card
# What Actually Happened
got two, along with everything else in the stupid thing

### LINKED ISSUE #29462: Gee Timmy, how come centcom sent you two shards.
(WEB REPORT BY: mastercasual REMOTE: 206.221.180.138:7777)
# Revision
648176ce0040d63f21f845a43f922bfbca856fb3

# Description
Two shards are spawning in the crates at engineering now a lot more frequently than usual 
# Steps to Reproduce
open a shard crate
# What you Expected
one shard
# What Actually Happened
two big old piece of cheese come out of the box

### PR DESCRIPTION
**Pull request by DeityLink**
Fixes #29462
Fixes #28744

:cl:
* bugfix: Fixed supermatter shard crates (and other crates potentially) spawning their content twice. (mastercasual)

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 1]
```diff
@@ -66,7 +68,14 @@
 
 /obj/structure/closet/initialize()
 	..()
-	spawn_contents()
+	if (!time_initialized_at)
+		time_initialized_at = world.time
+		spawn_contents()
+	else
+		//haha, so you'd like to initialize twice huh? you got some explaining to do kid.
```
Comment (boy2mantwicethefam) [inline code/game/objects/structures/crates_lockers/closets.dm:75]: Make this more aggressive

Comment (DeityLink) [reply code/game/objects/structures/crates_lockers/closets.dm:75]: I originally wrote "gotchu bitch"

Comment (boy2mantwicethefam) [reply code/game/objects/structures/crates_lockers/closets.dm:75]: You don't understand, you have to make it sound a lot more like someone who is smug about it. Someone whose style of speech is so aggravating that you want to cave their face in with the heel of your boot, something that the map-bot guy would say, something that gets under the skin like the "wholesome" users that are actually incredibly toxic in rule-heavy places, something that is proceeded with heavy smile emoji usage, something like that.

Comment (Eneocho) [reply code/game/objects/structures/crates_lockers/closets.dm:75]: >something that the map-bot guy would say

kek

**Pull request merged**